### ISSUE TITLE
Is there any way to reset BIOS settings?

### ISSUE THREAD

**Issue by p0wnystark** [sundarnagarajan/rdp-thinbook-linux#51]

**Comment by sundarnagarajan** [comment]
What do you mean? Reset BIOS settings using command line? No

**Comment by p0wnystark** [comment]
No, not by command line. I can't find a CMOS battery to reset the BIOS settings. And the thinbook is refusing to post. Do you know any hardware method to reset the BIOS?
(I don't know if i can ask this question here. I'm really sorry if asked this question at the wrong place.)

**Comment by sundarnagarajan** [comment]
Hmm... I haven't actually OPENED up any of the RDP Thinbooks to be able to answer this question properly.
I had received a photo of the innards of the 14-inch device in 2017, but I seem to have deleted it.
I would suggest you contact RDP directly.
